centaur/src/main/resources/standardTestCases/local_bourne/local_bourne.wdl

```diff
@@ -0,0 +1,20 @@
+task shell {
+  command {
+    # The SHELL variable isn't set in /bin/sh on alpine, busybox or ubuntu so there is this abomination instead.
+    ps auxwww | head -n 2 | tail -n 1 | cut -c22-28
```

Comment:
Is this to discover which shell is running? I beleive this is more reliable: 
sh -c 'ps -p $$ -o ppid=' | xargs -i readlink -f /proc/\{\}/exe
https://askubuntu.com/a/1022440/382239

Reply:
Thanks for the link! Turns out `busybox` is so slim that `ps` doesn't even support `-p`. https://busybox.net/BusyBox.html#ps

Out of curiosity I tried scanning through one of the POSIX standard versions and apparently `ps -p` might technically be required: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/ps.html

I guess this [comment](https://busybox.net/BusyBox.html#description) leaves a little wiggle room?
> BusyBox provides a fairly complete POSIX environment

Reply:
Ah it is in a busybox environment I did not realise. Ah well. So we just need a solution more portable that posix...

Reply:
@EvanTheB, I doubt that such a solution exists. If you were unable to locate one, do you mind filing an issue with Busybox? If you were, are you able to share it with me?